target/linux/ipq40xx/files-4.19/arch/arm/boot/dts/qcom-ipq4018-dlink-dap-2610.dts

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019-ap.dk01.1.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "D-Link DAP 2610";
+	compatible = "dlink,dap-2610";
+
+	soc {
+		edma@c080000 {
+			qcom,num_gmac = <1>;
+		};
+	};
+
+	aliases {
+		led-boot = &power;
```

Comment:
The [manual of the DAP-2610](ftp://ftp.dlink.de/dap/dap-2610/documentation/DAP-2610_man_reva_Manual_en.pdf) has the following to say about the LED - Indicators.

"Solid Red - Indicates the access point has malfunctioned."
"Blinking Red - This LED will blink during boot-up."
"Solid Green - Indicates   that   the   DAP-2610   is   working   properly"

Based on this description, it should be possible to mimic the same LED behavior with OpenWrt.
For example, if you add a label to the red led (i.e. led_redstatus) below and change the led-failsafe and led-boot aliases to:

led-boot = &led_redstatus;
led-failsafe = &led_redstatus;

the AP should behave like the DAP-2610 manual states (which is a plus).

Reply:
Sure. I was unsure what was the proper OpenWrt way of using the LEDs. Imitating  the factory behaviour works good.